packages/vest/src/core/createSuite/spec.js

```diff
@@ -94,6 +96,23 @@ describe('Test createSuite module', () => {
       expect(getSuite(suiteId)).toMatchSnapshot();
     });
 
+    it('Should be able to get the suite from the result of createSuite', () => {
+      const testsCb = jest.fn();
+      const suiteId = 'test_get_suite';
+      expect(createSuite(suiteId, testsCb).get()).toBe(get(suiteId));
+    });
+
+    it('Should be able to reset the suite from the result of createSuite', () => {
+      const suiteId = 'test_reset_suite';
+      const testSuite = createSuite(suiteId, () => {
+        dummyTest.failing('f1', 'm1');
+      });
+      testSuite();
+      expect(get(suiteId).hasErrors()).toBe(true);
+      testSuite.reset();
+      expect(get(suiteId).hasErrors()).toBe(false);
```

Comment:
I would also add another more explicit check here 
```suggestion
      testSuite();
      expect(get(suiteId).hasErrors()).toBe(true);
      expect(get(suiteId).testCount).toBe(1);
      testSuite.reset();
      expect(get(suiteId).hasErrors()).toBe(false);
      expect(get(suiteId).testCount).toBe(0);
```